library/src/main/kotlin/com/chuckerteam/chucker/internal/support/SpanTextUtil.kt

```diff
@@ -0,0 +1,132 @@
+package com.chuckerteam.chucker.internal.support
+
+import android.graphics.Color
+import android.text.SpannableStringBuilder
+import android.text.Spanned
+import android.text.style.ForegroundColorSpan
+import androidx.core.text.isDigitsOnly
+import com.google.gson.JsonElement
+import com.google.gson.JsonParser
+import com.google.gson.JsonSyntaxException
+
+
+public class SpanTextUtil {
+    public companion object {
+        private val JSON_KEY_COLOR = Color.parseColor("#8B0057")
+        private val JSON_STRING_VALUE_COLOR = Color.parseColor("#2F00FF")
+        private val JSON_DIGIT_AND_NULL_VALUE_COLOR = Color.parseColor("#E84B31")
+        private val JSON_SIGN_ELEMENTS_COLOR = Color.parseColor("#474747")
```

Comment:
Can we get those not hardcoded but moved to resource + have a `-night` variants with brighter and more pale colors?

That's how it looks like on dark mode:
![Screenshot_1677346348](https://user-images.githubusercontent.com/3001957/221371248-f84a4ab3-b71c-4561-8539-28762d3f806b.png)